core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -7382,6 +7382,25 @@ private void checkLiteral2(String expression, String expected) {
     sql(sql6).ok(expected6);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6116">[CALCITE-6116]
+   * Add EXISTS function (enabled in Spark library)</a>. */
+  @Test void testExistsFunctionInSpark() {
```

Comment:
also, a test with an empty array

Reply:
Thanks for your review! You are correct that using the `any` type is inaccurate, and it would be better to use the element type of the array for type validation. I will fix it later.